target/linux/ramips/dts/mt7621_cudy_wr1300-v2.dts

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "cudy,wr1300-v2", "mediatek,mt7621-soc";
+	model = "Cudy WR1300 v2";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_sys: sys {
+			label = "green:sys";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "green:wps";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <80000000>;
+		m25p,fast-read;
```

Comment:
It's better to use 50MHz spi frequency and drop fast-read, we've got some bad feedbacks not long ago. https://github.com/openwrt/openwrt/commit/857ea3f690aba8513b356926d9c430adafc7c50b https://github.com/openwrt/openwrt/commit/17e690017d76c49070bc99d6a376b1926259c5ff

Reply:
On this point I do not agree very much. It is true that certain router models are giving errors if you leave those high frequencies, but in this particular model I have not seen this behavior. In my opinion I think it is best to leave it at the current value and if any user has problems in the future lower this value. Lowering the value because someone might have problems without having evidence of it (with this particular model) I don't think it is the best option, especially considering that it is a trivial change that can be applied at any time.